presto-parquet/src/main/java/com/facebook/presto/parquet/cache/MetadataReader.java

```diff
@@ -219,11 +219,15 @@ private static void readTypeSchema(Types.GroupBuilder<?> builder, Iterator<Schem
     public static org.apache.parquet.column.statistics.Statistics<?> readStats(Statistics statistics, PrimitiveTypeName type)
     {
         org.apache.parquet.column.statistics.Statistics<?> stats = org.apache.parquet.column.statistics.Statistics.getStatsBasedOnType(type);
+        //manual initial num_nulls
+        stats.setNumNulls(-1);
```

Comment:
According to https://www.javadoc.io/doc/org.apache.parquet/parquet-column/1.10.0/org/apache/parquet/column/statistics/Statistics.html#isNumNullsSet--, there is a method that can tell if numNulls is set or not. Why should we set it to -1?
```
public boolean isNumNullsSet()

Returns:
    whether numNulls is set and can be used 
```